src/nvim/tui/input.c

```diff
@@ -209,7 +209,8 @@ static void forward_mouse_event(TermInput *input, TermKeyKey *key)
     }
   } else if (ev == TERMKEY_MOUSE_DRAG) {
     len += (size_t)snprintf(buf + len, sizeof(buf) - len, "Drag");
-  } else if (ev == TERMKEY_MOUSE_RELEASE) {
+  } else {
+    assert(ev == TERMKEY_MOUSE_RELEASE);
```

Comment:
I would rather make that `switch()` with `case TERMKEY_MOUSE_UNKNOWN: { assert(false); }` and other cases from this if()/elseif()/… At least this way we will have a warning here if termkey adds more mouse events.

Reply:
I like that idea, the way I have it now seems a bit off. Only problem I see is that on line 174 it's already been verified that `ev` can only be `TERMKEY_MOUSE_RELEASE`, `TERMKEY_MOUSE_DRAG`, or `TERMKEY_MOUSE_PRESS`. If I change it to a switch statement with `case TERMKEY_MOUSE_UNKNOWN: { assert(false); }` PVS will probably complain about unreachable code.

Reply:
Nevermind, PVS is okay with the `switch`. The `default` statement is still unreachable though, that okay?

Reply:
No `default` for enums, when enums happen to receive new value such switch()es need to be reviewed which will only happen without default.

Reply:
Gotcha, makes sense now